hydromt_sfincs/workflows/flwdir.py

```diff
@@ -117,11 +121,14 @@ def river_boundary_points(
         gdf_riv = gdf_riv.to_crs(region.crs).clip(region.unary_union)
         # filter river network based on uparea and length
         if "uparea" in gdf_riv.columns:
-            gdf_riv = [gdf_riv["uparea"] >= river_upa]
+            gdf_riv = gdf_riv[gdf_riv["uparea"] >= river_upa]
         if "rivlen" in gdf_riv.columns:
             gdf_riv = gdf_riv[gdf_riv["rivlen"] > river_len]
-
+        if reverse_river_geom:
+            geometry_column = gdf_riv.geometry.name
+            gdf_riv[geometry_column] = gdf_riv[geometry_column].reverse()
     dx = res / 5 if inflow else -res / 5
```

Comment:
Yes true, that would give the same results. The reason I did it this way was mainly that it would be more clear what is happening when reading the code, but your suggestion would indeed be simpler and faster

Reply:
I've added some comments to clarify.